examples/agent-sandbox-rl/agent_sandbox_rl/sizing.py

```
@@ -59,19 +69,87 @@ def recommend_window(
+    *,
+    avg_image_gb: float | None = None,
+    usable_disk_gb: float | None = None,
+    buffer: int = 0,
+    per_task: bool = False,
+    nodes: int = 1,
+) -> int:
+  """Window for the pipelined (double-buffered) strategy.
+
+  The pipeline keeps up to **two** windows resident at once, so halve the
+  concurrency window to keep peak ~ ``max_concurrent``; then, if disk hints are
+  given, cap by disk with a 2× factor (each cap applies its factor exactly once).
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)